### DISCUSSION TITLE
Richiesta di assistenza per l'aggiunta di un tutor PCTO sul Registro Elettronico

### DISCUSSION THREAD

**Discussion by robertopadovesi** [iisgiua/giuaschool#753] [Richieste di aiuto] [answered]
Gentile Team di Supporto,

Vi scrivo per chiedere assistenza in merito all'inserimento di un tutor PCTO (Percorsi per le Competenze Trasversali e l'Orientamento) all'interno del nostro registro elettronico. Attualmente, presso la nostra scuola, stiamo organizzando i percorsi PCTO per gli studenti del triennio e abbiamo necessità di integrare le informazioni relative al tutor direttamente nel sistema gestionale.

In particolare, il nostro obiettivo è poter assegnare un tutor per ciascuno dei percorsi PCTO attivati per i diversi indirizzi di studio. Questa figura sarà responsabile della supervisione delle attività di orientamento e delle competenze trasversali. Vorremmo, quindi, sapere quali siano i passaggi da seguire per configurare correttamente il tutor e collegarlo alle classi e ai gruppi di studenti a cui è assegnato.

Un aspetto importante per noi è la possibilità di inserire non solo il nome del tutor, ma anche i suoi contatti (email e telefono) e le ore di disponibilità settimanali, in modo che gli studenti e le famiglie possano facilmente accedere a tali informazioni tramite il registro elettronico. Vorremmo sapere se esiste un'apposita sezione per questa funzione o se dobbiamo creare dei campi personalizzati.

Inoltre, sarebbe utile conoscere le modalità per aggiornare o modificare i dati del tutor in corso d'anno, nel caso in cui ci siano variazioni. Vi chiediamo anche se è possibile collegare ogni tutor a specifici obiettivi PCTO, così da monitorare i progressi degli studenti rispetto a tali traguardi.

Un altro aspetto che ci interessa approfondire riguarda l'integrazione del tutor PCTO con la funzione di reportistica. Ci piacerebbe avere la possibilità di generare report personalizzati che evidenzino il contributo di ciascun tutor nel percorso di apprendimento degli studenti.

Vi chiediamo, inoltre, se vi siano delle linee guida specifiche da seguire per garantire la conformità ai requisiti di privacy nel trattamento dei dati del tutor e degli studenti. La nostra priorità è proteggere tutte le informazioni sensibili, nel rispetto delle normative vigenti.

Qualora fosse necessario un intervento tecnico, siamo a disposizione per concordare un incontro online o telefonico con il vostro team. L'integrazione del tutor PCTO è una componente essenziale per il nostro progetto educativo e desideriamo procedere nel miglior modo possibile.

Vi ringraziamo anticipatamente per la disponibilità e attendiamo vostre indicazioni per poter completare questa procedura in modo efficace e sicuro. Ogni suggerimento o guida operativa sarà per noi di grande aiuto.

Restiamo a disposizione per qualsiasi ulteriore chiarimento. In attesa di un vostro gentile riscontro, vi porgo i miei più cordiali saluti.

**Comment by trinko** [comment, marked as answer]
Mi dispiace, ma attualmente il registro non ha alcuna gestione per il PCTO.

**Reply by robertopadovesi** [reply]
Se inserisco la materia PCTO e le assegno ai docenti, si può fare? 
@trinko

**Reply by trinko** [reply]
> Se inserisco la materia PCTO e le assegno ai docenti, si può fare? @trinko

Non risolvi tutte le problematiche, ma può essere un inizio.

Un problema che però rimane è che devi prevedere che il tutor lavori con l'intera classe: in questo  modo considerando le ore della materia PCTO ed eventuali assenze degli alunni avresti almeno un quadro delle attività svolte.

La gestione potrebbe funzionare se tutte le ore di PCTO vengono svolte in orario curricolare. Se devi considerare anche quelle extracurricolari, allora le cose si complicano perché dovresti:
- aggiungere alle normali ore di lezioni anche le ore extracurricolari (es. lezioni 8.30-13.30, extracurricolare 15-19);
-  gestire eventuali assenze al PCTO come uscite anticipate (es., l'alunno svolge regolarmente la lezione la mattina, ma poi non partecipa al pomeriggio al PCTO).
